cross/tt-rss/Makefile

```diff
@@ -1,11 +1,11 @@
 PKG_NAME = tt-rss
-PKG_VERS = 17.4
+PKG_VERS = 67f02e2aa7
```

Comment:
Ok so here is the thing: tt-rss is no longer versioned.
And the archive page is empty. So the only real option we have now is to perform a git checkout and use a given hash.